src/printutils.cc

```diff
@@ -71,6 +76,9 @@ void PRINT_NOCACHE(const std::string &msg)
 			outputhandler(msg, outputhandler_data);
 		}
 	}
+	if(stop && OpenSCAD::hardwarnings){
+		throw HardWarningException("");
```

Comment:
It's probably good style to provide some information in the exception, like the error message?

Reply:
you are right. Under normal circumstances it is redundant, but when the exception slips trough the net, the message is at least outputted on standard output, allowing to search for the hole in the net.